### PR TITLE
S rum review prep

### PR DESCRIPTION
**Pull request by HamishBrownPFR**
working on #7869

### CODE REVIEW

**Comment by HamishBrownPFR** [comment]
retest this please jenkins

**Comment by HamishBrownPFR** [comment]
@hol353 please review and merge if OK.  @jbrider once this is merged the SCRUM protype in master will be up to date

File: Models/PMF/SimplePlantModels/ScrumCropInstance.cs [hunk 1]
```diff
@@ -556,36 +675,44 @@ public DateTime GetHarvestDate(DateTime start, double HarvTt, double BaseT, doub
             }
             return d;
         }
+
+        /// <summary>Answer to a binary or polar question.</summary>
+        public enum YesNoAnswer
+        {
```
Comment (hol353) [inline Models/PMF/SimplePlantModels/ScrumCropInstance.cs:681]: Is this used? There is never a need for this. Just use a boolean.

Comment (HamishBrownPFR) [reply Models/PMF/SimplePlantModels/ScrumCropInstance.cs:681]: @rcichota I think this relates to the change you made.  Can you please respond to this

Comment (rcichota) [reply Models/PMF/SimplePlantModels/ScrumCropInstance.cs:681]: I added this to make clearer in the GUI which way the answer goes (used to tell whether the model should consider water stress or not). I think that having just a tick can be ambiguous at times... If needed I can easily remove the enum and use the boolean/tick box

Comment (hol353) [reply Models/PMF/SimplePlantModels/ScrumCropInstance.cs:681]: Tick boxes are simpler. To make them less ambiguous, frame the text beside the tick box as a question.

Comment (rcichota) [reply Models/PMF/SimplePlantModels/ScrumCropInstance.cs:681]: Ok I have reverted this change to a simple tick box. I still think that a yes/no marker is less ambiguous than a tick mark (as can be seen below). And indeed, it helps to force the coder to make the input into a question. But I also recognise it is a very small thing and I am not getting into a fight for it...

image
image

**Comment by HamishBrownPFR** [comment]
@hol353 can you please recheck and merge if happy.  @jbrider sorry for the hold up

**Pull request merged**